CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### Bugfix
 
+- fix universal link @nileshgulia1s
```

Comment:
```suggestion
- fix universal link when no item content obj passed @nileshgulia1
```
I would add a bit more explanation here, just to make it clearer. The PR title is fine imho

Reply:
I fixed the typo in changelog and made it more accurate. Also added `isRequired` propType to `item[@id]`